wasmer2/bridge2.go

```diff
@@ -3,6 +3,7 @@ package wasmer2
 // #cgo LDFLAGS: -Wl,-rpath,${SRCDIR} -L${SRCDIR}
 // #cgo linux,amd64 LDFLAGS:-lvmexeccapi
 // #cgo darwin,amd64 LDFLAGS:-lvmexeccapi
+// #cgo darwin,arm64 LDFLAGS:-lvmexeccapi_arm
```

Comment:
Didn't change the name to `..._arm64`, since:

https://github.com/multiversx/mx-vm-executor-rs/blob/master/Makefile#L20